dts/bindings/i2c/atmel,sam-i2c-twim.yaml

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2020 Gerson Fernando Budke <nandojve@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Atmel SAM Family I2C (TWIM) node
+
+compatible: "atmel,sam-i2c-twim"
+
+include: i2c-controller.yaml
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: true
+
+    peripheral-id:
+      type: int
+      required: true
+      description: peripheral ID
+
+    std-clk-slew-lim:
+        type: int
+        description: Slew limit of the TWCK output buffer
+        default: 0
```

Comment:
> Are these values the power-up default for the peripheral?

Yes, are the power-up/reset defaults. However, the right semantics is required true for all props with default value. In this case, I prefer drop all default attributes and force user to set-up values. The parameters are to tune the slope on both SCL/SDA lines. At end, user should put lines on scope and run some tests to make sure everything is ok.

> Also the description should give a clue as to what values are expected.

Tried improve a little bit since docs not collaborate.

> Does the slew limit correspond to a HAL defined constant? It may be that an enumeration is a better type.

Unfortunately no. I took look at electrical characteristics to get some clue. I found the values based on simulation. With that I created the enum. The slew prop is a complete secret (I don't know the why). At end, user needs run tests on scope to tune the lines. I created an int enum just to limit the values.